src/components/Header/Logouticon.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { PropTypes } from "prop-types"
+import { NavLink } from 'react-router-dom'
+
+import {
+    ExitIconWrapper,
+    MenuInitiaLetterCircle,
+    MenuInitiaLetter,
+    MenuName
+} from "./Header.styled"
+import ExitIcon from "../../assets/image/Header-exit-icon.svg"
+
+const Logouticon = (props) => {
+    const logoutAction = () => { props.logoutProps() }
```

Comment:
I dont see a reason for this function, you can use logout directly
PS and dont call it logoutProps, just `logout`, or if you need to have a logout function and logout action creator you can call it  `logout `and `logoutAction`

Reply:
I removed the props passing from this file and the logout click event from the parent. Now this function and event are in this file))
![Picture1](https://user-images.githubusercontent.com/96341034/189674044-e972779b-7222-430d-bfa7-339144cada85.png)